cmudb/extensions/tscout/main.c

```diff
@@ -0,0 +1,224 @@
+#include <time.h>
+
+// clang-format off
+// Extension magic.
+#include "postgres.h"
+#include "fmgr.h"
+// clang-format on
+
+#include "commands/createas.h"
+#include "commands/explain.h"
+#include "optimizer/planner.h"
+#include "parser/parsetree.h"
+#include "utils/builtins.h"
+
+#include "../../../src/include/tscout/marker.h"
+#include "operating_unit_features.h"
+
+PG_MODULE_MAGIC;
+void _PG_init(void);
+void _PG_fini(void);
+
+static void ExplainOneQueryWrapper(Query *query, int cursorOptions, IntoClause *into, ExplainState *es,
+                                   const char *queryString, ParamListInfo params, QueryEnvironment *queryEnv);
+static void WalkPlan(Plan *plan, ExplainState *es);
+static void explainXs(Plan *node, ExplainState *es);
```

Comment:
Please be consistent with function naming. As a rough style guide, in the old NoisePage project, we preferred `FunctionNamesToBeLikeThis()`.

```suggestion
static void ExplainFeatures(Plan *node, ExplainState *es);
```